src/window.rs

```diff
@@ -344,6 +346,15 @@ impl WindowBuilder {
         self
     }
 
+    /// Sets whether the background of the window should be blurred by the system.
+    ///
+    /// The default is `false`.
```

Comment:
Ditto about platform specifics, but you can point to `Window::set_blur` from here saying that platform specific docs are there.

Reply:
It's actually not resolved, since you don't link.